pennylane/ops/op_math/condition.py

```diff
@@ -112,6 +114,16 @@ def cond(condition, true_fn, false_fn=None, elifs=()):
     will be captured by Catalyst, the just-in-time (JIT) compiler, with the executed
     branch determined at runtime. For more details, please see :func:`catalyst.cond`.
 
+    When used with :func:`~.pennylane.capture.enabled`, this function allows for general

```

Comment:
Given the capture project is experimental and internal at this point, I don't want to confuse and complicate this docstring for users.  Maybe we can have a little toggle at the bottom of the docstring for "Experimental program capture considerations" or something.  The part of the of the docstring above `Args:` should usually be just 1-2 sentences. 

 So yes, we have added a bunch of extra information here for qjit, but I don't think this is the place for program capture information.

Reply:
Thanks for the suggestions. I agree with pretty much all the points. I left some comments so that it is easier to debug and is not a bunch of spaghetti.

As for this point, I moved the discussion to the last expandable section before `Args`, and it's just a couple of sentences.